omod/src/main/java/org/openmrs/module/visitdocumentsui/page/controller/VisitDocumentsPageController.java

```diff
@@ -46,6 +46,7 @@ public void controller(
 		jsonConfig.put("visit", visitWrapper == null ? null : convertVisit(visitWrapper.getVisit()));
 
 		jsonConfig.put("contentFamilyMap", getContentFamilyMap());
+		jsonConfig.put("associateWithVisitAndEncounter", context.associateWithVisitAndEncounter());
```

Comment:
I'm not crazy with this long-winded name "associatedWithVisitAndEncounter"... certainly open to better ideas!